### ISSUE TITLE
Missing Bulkrax config line

### ISSUE THREAD

**Issue by kirkkwang** [samvera/hyku#1850]
### Descriptive summary

Bulkrax config template was not updated to include sidebar config for Hyrax 3 during upgrade.

### Rationale

Once Bulkrax is updated again to include changes from https://github.com/samvera-labs/bulkrax/pull/668, menu options will not appear dynamically until the configuration is updated.

### Related work

https://github.com/samvera/hyku/blob/main/config/initializers/bulkrax.rb
should look like
https://github.com/samvera-labs/bulkrax/blob/main/lib/generators/bulkrax/templates/config/initializers/bulkrax.rb

### Recommendation

Basically, just copy the code from Bulkrax over to Hyku.
